packages/ethereum-contracts/test/foundry/superfluid/Superfluid.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity 0.8.19;
+
+import "../FoundrySuperfluidTester.sol";
+import { SuperToken } from "../../../contracts/superfluid/SuperToken.sol";
+import {
+    SuperTokenV1Library
+} from "../../../contracts/apps/SuperTokenV1Library.sol";
+import {
+    ISuperAgreement
+} from "../../../contracts/interfaces/superfluid/ISuperAgreement.sol";
+import {
+    ISuperfluid
+} from "../../../contracts/interfaces/superfluid/ISuperfluid.sol";
+import { AgreementMock } from "../../../contracts/mocks/AgreementMock.sol";
+
+contract SuperfluidIntegrationTest is FoundrySuperfluidTester {
+    using SuperTokenV1Library for SuperToken;
+
+    uint32 private constant _MAX_NUM_AGREEMENTS = 256;
+    uint32 private constant _NUM_AGREEMENTS = 2;
```

Comment:
a bit fragile logic, when GDA included it would be 3, it seems the test would fail that's good (but if not, then it could become a dummy test case)

Reply:
But indeed `_agreementClasses` is current not exposing .length or anything, something to think about if you want to be perfectionist here.

Reply:
it will break, the original test broke in the GDA branch when running this